Add migration to fix coretime state (#458)
polkadot-fellows/runtimes: CHANGELOG.md

### Fixed

- Chain-spec generator: propagate the `on_chain_release_build` feature to the chain-spec generator. Without this the live/genesis chain-specs contain a wrongly-configured WASM blob ([polkadot-fellows/runtimes#450](https://github.com/polkadot-fellows/runtimes/pull/450)).

> - Adds a migration to the Polkadot Coretime chain to fix an issue from the initial Coretime migration. [polkadot-fellows/runtimes#458](https://github.com/polkadot-fellows/runtimes/pull/458)

Reviewer: ```suggestion
- Adds a migration to the Polkadot Coretime chain to fix an issue from the initial Coretime migration. ([polkadot-fellows/runtimes#458](https://github.com/polkadot-fellows/runtimes/pull/458))
```

### Fixed

- Chain-spec generator: propagate the `on_chain_release_build` feature to the chain-spec generator. Without this the live/genesis chain-specs contain a wrongly-configured WASM blob ([polkadot-fellows/runtimes#450](https://github.com/polkadot-fellows/runtimes/pull/450)).

> - In the initial Coretime migration the leases were off due to the lease offset on
> Polkadot. [polkadot-fellows/runtimes#458](https://github.com/polkadot-fellows/runtimes/pull/458)
> fixes this via another storage migration on the Coretime chain.

Reviewer: ```suggestion
- Adds a migration to the Polkadot Coretime chain to fix an issue from the initial Coretime migration. [polkadot-fellows/runtimes#458](https://github.com/polkadot-fellows/runtimes/pull/458)
```
